src/main/scala/io/qbeast/spark/delta/QbeastMetadataSQL.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Qbeast Analytics, S.L.
+ */
+package io.qbeast.spark.delta
+
+import io.qbeast.core.model.{CubeId, NormalizedWeight, Weight}
+import io.qbeast.spark.utils.TagColumns
+import org.apache.spark.sql.Column
+import org.apache.spark.sql.expressions.UserDefinedFunction
+import org.apache.spark.sql.functions.{col, struct, udf}
+
+object QbeastMetadataSQL {
+  val weight: UserDefinedFunction = udf((weight: Int) => Weight(weight))
+
+  val norm: UserDefinedFunction = udf((mw: Weight, elementCount: Long, desiredSize: Int) =>
```

Comment:
Could normWeight or normalizedWeight be a better name?

Reply:
Yep or normalizeWeight, since it's a method